src/libertem/api.py

```diff
@@ -451,6 +543,12 @@ def run(self, job: Union[Job, BaseAnalysis], roi=None):
         ----------
         job
             the job or analysis to run
+
+        Returns
+        -------
+        result : numpy.ndarray or libertem.analysis.base.AnalysisResultSet
+            Running a Job returns a :class:`numpy.ndarray`, running
+            an Analysis returns a :class:`libertem.analysis.base.AnalysisResultSet`
```

Comment:
Maybe also add: "see the matching `create_*_analysis` function for documentation of the specific `AnalysisResultSet` fields"